WooCommerce/Classes/ViewRelated/FilterTabBar/TabbedViewController.swift

```diff
@@ -0,0 +1,105 @@
+import UIKit
+
+// NOTE: this file is adapted from WPiOS.
```

Comment:
What do you think of adding a link to the source version?

```suggestion
// NOTE: this file is adapted from WPiOS.
//
// https://github.com/wordpress-mobile/WordPress-iOS/blob/67a5d840eeb9ac027404ad71ed246c645de33687/WordPress/Classes/ViewRelated/Reader/Manage/TabbedViewController.swift
```

Reply:
Good point! added in https://github.com/woocommerce/woocommerce-ios/pull/8942/commits/27adc006d64aaef1db315022feb695c18bb9c640